src/neptune/cli/clear.py

```diff
@@ -87,11 +89,17 @@ def remove_data(
         ClearRunner.remove_containers(offline_containers_paths)
         ClearRunner.remove_containers(unsynced_containers_paths)
 
+    @staticmethod
+    def remove_synced_data(container_manager: ContainersManager, synced_containers: Sequence["ApiExperiment"]) -> None:
+        synced_containers_paths = [container_manager.resolve_async_path(container) for container in synced_containers]
+        ClearRunner.remove_containers(synced_containers_paths)
+
     @staticmethod
     def remove_containers(paths: List[Path]) -> None:
         for path in paths:
-            try:
-                shutil.rmtree(path)
-                logger.info(f"Deleted: {path}")
-            except OSError:
-                logger.warn(f"Cannot remove directory: {path}")
+            if path.exists():
+                try:
+                    shutil.rmtree(path)
```

Comment:
What do you think about make it the same as https://github.com/neptune-ai/neptune-client/blob/fe39583f7aa580fa040f5b709946abcdeedae9ca/src/neptune/cli/clear.py#L64? (``ignoreErrors=True`` or left only one implementation)

Reply:
That's completely different thing. Sync that's an Experiment (run, model etc.) created in `sync` mode (every operation = immediate request to server) vs async that wasn't cleaned up properly (mode `async` that synced all the operations but there are leftovers)

Reply:
In that particular case, I meant diff at error handling - in one case we are using the bultin flag to suppress and in another, we are using try...except -> what do you think about make it the same?

Reply:
Hmmmm. We probably wanted here (I've copied and pasted it from some other cleanups) to be more explicit. With `sync` we're not losing anything etc and that's a "bulk" delete of all containers so quite different usage.